### DISCUSSION TITLE
Hackathon: 2024-10-01

### DISCUSSION THREAD

**Discussion by asteiker** [earthaccess-dev/earthaccess#831] [Hack days]
Reporting out on earthaccess hack days.

Please create a new comment (at the very bottom, use the text box with a button that says "comment", not "reply") to write about what you did today!

Please include links to GitHub issues or pull requests relevant to your work, and use those issues or pull requests as the "source of truth" for documenting the work; this makes future readers jobs easier :) It's OK to duplicate some information while writing a summary here!

Use the "reply" feature to have a discussion under any given comment.

**Comment by mfisher87** [comment]
@nikki-t and I went through the release process together, released 0.11.0 :tada: (https://github.com/nsidc/earthaccess/releases/tag/v0.11.0), updated the docs (#834), and identified a gap in the docs (https://github.com/nsidc/earthaccess/issues/833)

**Comment by JessicaS11** [comment]
See https://github.com/nsidc/earthaccess/discussions/794#discussioncomment-10835776 for @asteiker's and my notes on some ideas for the upcoming Hackday after the NSIDC UWG meeting next week
